types/api/mongo-query.ts

```diff
@@ -1,8 +1,8 @@
 export interface MongoQuery {
 
-    count : boolean
-    length: Number
-    skip  : Number
-    sort  : String
+    count? : boolean
+    length?: Number
+    skip?  : Number
+    sort?  : Object
```

Comment:
or Record<string, number>
```suggestion
    sort?  : Record<string, number>
```